src/blueprints/plugin.py

```
@@ -305,46 +305,72 @@ def save_plugin_settings():
+        return _save_plugin_settings_common(
+            plugin_id=plugin_id,
+            plugin_settings=plugin_settings,
+            device_config=device_config,
+            playlist_manager=playlist_manager,
+        )
+    except Exception:
+        logger.exception("Error saving plugin settings (alias)")
+        return json_error("An internal error occurred", status=500)
+
+
+def _save_plugin_settings_common(plugin_id, plugin_settings, device_config, playlist_manager):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `_save_plugin_settings_common`

(ANN202)